python/common/org/python/types/Range.java

```diff
@@ -118,4 +121,50 @@ public Range(org.python.Object start, org.python.Object stop, org.python.Object
         }
     }
 
+    @org.python.Method(
+        __doc__ = "Implement __len__(self)."
+    )
+    public org.python.types.Int __len__() {
```

Comment:
OK. It looks like \_\_repr\_\_ is returning a String, and \_\_iter\_\_ is returning an Iterable. Should they also have the return type org.python.Object?